.dependency_license

```diff
@@ -31,6 +31,7 @@ RobotoSlab-Regular\.ttf$, Apache
 \.config(\.example)?$, Apache
 /\.bowerrc$, Apache
 /\.jshintrc$, Apache
+/\.jshintignore$, Apache
```

Comment:
No need to ignore this file. `.jshintignore` can have the Apache license at the top, prefixed with hashes (e.g., [like we do for dockerfiles](https://github.com/apache/trafficcontrol/blob/40934f3213/infrastructure/cdn-in-a-box/traffic_ops/Dockerfile#L1-L16)).